Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'mysql2', '>= 0.4.4'
 gem 'puma', '~> 4.1'
 gem 'redis', '~> 4.0'
 gem 'aws-sdk-s3', '~> 1'
+gem "anycable-rails", "~> 0.6"#, "~> 1.0.0.rc1"
```

Comment:
@ErikSchierboom @kntsoriano Read about AnyCable.

Reply:
I've been using AnyCable for our virtual club!

Reply:
Sounds great! AnyCable is used to optimize the performance of WebSockets, is that correct?

Reply:
@ErikSchierboom Yes. It basically removes Ruby from the server/connection bit and handles that with something more speedy (Go). It then connects to the Rails app via Redis pub/sub to work out what needs sending where.